### PR TITLE
Readme updates for helm

### PR DESCRIPTION
**Pull request by SimoneLazzaris**
added readme instructions 
added a Makefile in helm directory to build the helm package

### CODE REVIEW

**Review by byo — DISMISSED** [review]
LGTM (with one minor comment)!

File: helm/Chart.yaml [hunk 1]
```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: immudb
 description: The immutable database
 type: application
-version: 0.1.0
+version: 1.1.0
```
[the remark this answers is not in the document]
Comment (SimoneLazzaris) [reply helm/Chart.yaml:5]: I wanted to match the version of the application itself, don't know if it makes sense

[the remark this answers is not in the document]
Comment (byo) [reply helm/Chart.yaml:5]: ah, right, we could update this too, although in most cases I can think of I've seen version and appVersion different, I'd say that we should bump the version field only when the helm chart has some changes.

[the remark this answers is not in the document]
Comment (SimoneLazzaris) [reply helm/Chart.yaml]: Resolved in commit 94ccdffd, version set to 1.0.0

**Review by byo — DISMISSED** [review]

**Review by byo — APPROVED** [review]

**Pull request merged**